README.md

```diff
@@ -1,2 +1,22 @@
-# rebar3-nix-bootstrap
-Shim command to help bootstrap a rebar3 project on Nix
+Rebar3 Nix Bootstrap
+====================
+
+This is a very simply 'shim' project that exists to help rebar3 work
+in the context of Nix. Its purpose is to bootstrap a project so that
+rebar3 doesn't try to manage dependencies (that should be left to
+Nix).
+
+This does the following work.
+
+# It copies the hex registry to a place where rebar3 can find it
+# It copies any dependencies into the `_build` dir to a place where
+  rebar3 can find it.
+# It updates the `*.app.src` with the version that is available to
+  from the nix expression.
+
+Usage
+----
+
+Simply run `rebar3-nix-bootstrap` inside of a `nix-shell` with
+relevant Erlang support. It will figure things out and do the right
```

Comment:
Oh, sorry I was not clear enough. I meant adding an example like

    nix-shell -A erlangPackages.ibrowse.env --run "rebar3-nix-bootstrap"

Or even better, once we have rebar2nix command, something like:

    rebar2nix --shell > shell.nix
    nix-shell --run "rebar3-nix-bootstrap"

WDYT?